src/agent/main.cpp

```diff
@@ -218,13 +238,13 @@ int main(int argc, char *argv[])
         SuccessOrExit(ret = instance.Init());
 
 #if OTBR_ENABLE_OPENWRT
-        ot::BorderRouter::Ncp::ControllerOpenThread *ncpThread =
-            static_cast<ot::BorderRouter::Ncp::ControllerOpenThread *>(ncp);
+        ControllerOpenThread *ncpThread = reinterpret_cast<ControllerOpenThread *>(ncp);
```

Comment:
We need to rethink the inheritance structure.

Reply:
Let's discuss in the Openwrt issue.